Jenkinsfile

```diff
@@ -14,22 +14,22 @@ pipeline {
 
   environment {
     DOCKER_HOST = "unix:///var/run/docker.sock"
-    HOSTED_GRAPHITE_ACCOUNT_ID = credentials('graphite_account_id')
-    HOSTED_GRAPHITE_API_KEY = credentials('graphite_api_key')
   }
 
   stages {
     stage('Maven Build') {
       steps {
-        sh 'docker pull govukpay/postgres:9.4.4'
-        sh 'mvn clean package'
+        script {
+          def long stepBuildTime = System.currentTimeMillis()
+
+          sh 'docker pull govukpay/postgres:9.4.4'
+          sh 'mvn clean package'
+          postSuccessfulMetrics("adminusers.maven-build", stepBuildTime)
```

Comment:
silly question, but does it definitely not get here on failure?

Reply:
Yes, did a test and added a mvn belinda step which failed
https://build.ci.pymnt.uk/job/pay-adminusers/view/change-requests/job/PR-165/12/console